src/basic/fileio.h

```diff
@@ -28,6 +28,8 @@
 #include "macro.h"
 #include "time-util.h"
 
+#define READ_FULL_BYTES_MAX 0x400000U
```

Comment:
it certainly makes sense to turn this into a `#define`, but I think we shouldn't export this as long as nothing needs it.